events/event_handler.go

```diff
@@ -160,6 +161,18 @@ func (fh *eventHandler) handleStatusEvent(body []byte) error {
 		"TaskStatus": task.TaskStatus,
 	}).Info("Got StatusEvent")
 
+	app, err := fh.marathon.App(task.AppID)
```

Comment:
I need to test it, this could double the traffic to Marathon for a deployment. Becouse new tasks will need to be registered and old will check if should be deregistered. 

Instead of asking marathon I'll prefer to ask consul if it has specific task and then deregister it. Third solution is to check error and log if it's not error that means there is no such task. If we want to deregister task and it's already gone we are done and this is not an error.